### ISSUE TITLE
timers continue when freezing operations

### ISSUE THREAD

**Issue by rtoma** [nolar/kopf#673]
## Long story short

Thanks for creating kopf. I am currently developing an operator that updates Egress NetworkPolicy resources to sync its `spec.egress[*].to[*].ipBlock.cidr` keys with A records for DNS hostnames.

This operator used timers (looking into daemons to follow DNS TTL). I run an operator in k8s with cluster peering. For development I start a local instance with --dev.

I see the operator in k8s detects the development instance:

```
Freezing operations in favour of [<Peer rtoma@devsystem/20210208145159/nz5: priority=666, lifetime=60, lastseen='2021-02-08T14:52:01.160123'>].
```

The development instance's log show:

```
Resuming operations after the freeze. Conflicting operators with the same priority are gone.
```

Still both operator instances show `Timer 'NAME' succeeded` messages. I searched the docs, but could not find anything.

## Description

<details><summary>The code snippet to reproduce the issue</summary>

```python
import kopf
```
</details>

<details><summary>The exact command to reproduce the issue</summary>

```bash
kopf run ...
```
</details>

<details><summary>The full output of the command that failed</summary>

```
```
</details>

## Environment

* Kopf version: 1.29.1
* Kubernetes version:  1.19.5
* Python version:  3.9.1
* OS/platform: k8s on pi4, local on arm mbp

<details><summary>Python packages installed</summary>

```
```
</details>

**Comment by nolar** [comment]
Indeed, that’s possible. And that is clearly a bug. Thanks for reporting — I’ll take a look if it can be fixed easily.

**Comment by nolar** [comment]
The fix is released in `kopf>=1.30.0rc1`.
